en/docs/get-started/about-this-release.md

```diff
@@ -56,6 +56,15 @@ It is now available to download from [here](https://github.com/wso2/product-apim
 - Defaulting code and configs to do with event based
 -->
 
+#### What has changed in this release
+
+##### Removed features and functionalities
+
+- Microgateway Labels
+
+      [Grouping APIs with Microgateway Labels](https://apim.docs.wso2.com/en/latest/learn/api-microgateway/grouping-apis-with-labels/)
```

Comment:
Shall we give more context in this regard please. If we are going to propose the user to use the vhost feature as an alternative we need to mention it here.

Reply:
updated